site/styles/_variables.scss

```diff
@@ -33,10 +33,11 @@ $container-max-widths: (
 // Body / Core
 // ------------------------------------
 
-$primary:         #35a0a8 !default;
-$secondary:       #4cd3dd !default;
-$body-bg:         #000 !default;
-$body-color:      #fff !default;
+$primary:         #4d97ff !default;
```

Comment:
I don't think we need to anything changed/added in this file.

Reply:
This is the new colour that design wants to have as default.